### PR TITLE
docs: node-status.md

### PR DESCRIPTION
**Pull request by HelloBug0**
This is the description of plugin node-status in Chinese, and I'd love to submit the document in lEnglish later.
This PR is relate with [issue#2379](https://github.com/apache/apisix/issues/2379)

### CODE REVIEW

**Comment by membphis** [comment]
CI failed, please take a look at the output: https://github.com/apache/apisix/pull/2615/checks?check_run_id=1347360640#step:11:51

**Review by HelloBug0 — COMMENTED** [review]
Change tabs to spaces

File: doc/zh-cn/plugins/node-status.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+
+# 目录
+- [**插件简介**](#插件简介)
```
Comment (membphis) [inline doc/zh-cn/plugins/node-status.md:22]: add a blank line

File: doc/zh-cn/plugins/node-status.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+- [**插件属性**](#插件属性)
+- [**插件接口**](#接口插件)
+- [**启用插件**](#启用插件)
+- [**测试插件**](#测试插件)
+- [**禁用插件**](#禁用插件)
+
+
+## 插件简介
+
+`node-status` 是APISIX的请求状态查询插件，返回基本的状态信息。
```
Comment (membphis) [inline doc/zh-cn/plugins/node-status.md:32]: need a space between Chinese and English

Comment (membphis) [reply doc/zh-cn/plugins/node-status.md:32]: and please fix other similar points

File: doc/zh-cn/plugins/node-status.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+
+## 插件接口
+
+插件增加接口`/apisix/status`，可通过 [interceptors](plugin-interceptors.md)保护该接口。
+
+
+## 启用插件
+
+1. 配置文件apisix/conf/conf.yaml的plugins list中配置node-status
```
Comment (membphis) [inline doc/zh-cn/plugins/node-status.md:47]: ditto

File: doc/zh-cn/plugins/node-status.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+
+启动APISIX之后，即可访问该插件提供的接口，获得基本的状态信息。
+
+2. 在创建route时添加插件node-status
+
+```
+$ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -i -d '
+{
+    "uri": "/route1",
+    "methods": ["PUT", "GET"],
```
Comment (membphis) [inline doc/zh-cn/plugins/node-status.md:67]: we can remove this line, it is not important

File: doc/zh-cn/plugins/node-status.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+    "upstream": {
+        "type": "roundrobin",
+        "nodes": {
+            "39.97.63.215:80:": 1
```
Comment (membphis) [inline doc/zh-cn/plugins/node-status.md:71]: we can use a local IP address, eg: `192.168.1.100:80`

**Review by membphis — COMMENTED** [review]
we can update the English version too

File: doc/zh-cn/plugins/node-status.md (hunk 4 above)
Comment (HelloBug0) [reply doc/zh-cn/plugins/node-status.md:67]: Thanks for your comments, and here you mean remove from line 61 to line 81?

File: doc/zh-cn/plugins/node-status.md (hunk 3 above)
Comment (Firstsawyou) [inline doc/zh-cn/plugins/node-status.md:47]: The file name of `conf.yaml` is incorrect, it should be `config.yaml`, right?

File: doc/zh-cn/plugins/node-status.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+        }
+    },
+    "plugins": {
+        "node-status":{}
+    }
+}'
+```
+
+发送该请求的前提是apisix/conf/conf.yaml中已经配置node-status，此时node-status插件对该请求处理无影响，所以一般不会将node-status插件设置到路由中。
```
Comment (Firstsawyou) [inline doc/zh-cn/plugins/node-status.md:80]: The file name of conf.yaml is incorrect, it should be config.yaml.

File: doc/zh-cn/plugins/node-status.md [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+| writing      | 当前正在写给客户端响应的连接数               |
+| handled      | 已经处理的连接总数,通常等于accepted          |
+| active       | 当前活跃的客户端连接数                       |
+| reading      | 当前正在读取请求头的连接数                   |
+| id           | APISIX uid信息，保存在apisix/conf/apisix.uid  |
+
+
+## 禁用插件
+
+1. 配置文件apisix/conf/conf.yaml的plugins list中删除node-status
```
Comment (Firstsawyou) [inline doc/zh-cn/plugins/node-status.md:116]: ditto.

**Review by Firstsawyou — CHANGES_REQUESTED** [review]

File: doc/zh-cn/plugins/node-status.md [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+
+
+## 插件属性
+
+插件属性为空。
+
+
+## 插件接口
+
+插件增加接口`/apisix/status`，可通过 [interceptors](plugin-interceptors.md)保护该接口。
```
Comment (spacewander) [inline doc/zh-cn/plugins/node-status.md:42]: Need a space after `interceptors.md)`

File: doc/zh-cn/plugins/node-status.md (hunk 6 above)
Comment (spacewander) [inline doc/zh-cn/plugins/node-status.md:80]: Do you mean `apisix/conf/config.yaml`? Also, need spaces between Chinese and English word.
Please fix other similar places in the PR.

File: doc/zh-cn/plugins/node-status.md (hunk 3 above)
Comment (HelloBug0) [reply doc/zh-cn/plugins/node-status.md:47]: yes, I will change it. Thanks~

File: doc/zh-cn/plugins/node-status.md (hunk 6 above)
Comment (HelloBug0) [reply doc/zh-cn/plugins/node-status.md:80]: OK, Thanks~

**Comment by HelloBug0** [comment]
I miss adding the commment when I commit the document at the third time, sorry about it.
I update this document according to the comments except [this](https://github.com/apache/apisix/pull/2615#discussion_r517034614). I will change it later after making sure about it.
I really appreciate all your comments.

File: doc/zh-cn/plugins/node-status.md (hunk 1 above)
Comment (membphis) [reply doc/zh-cn/plugins/node-status.md:23]: @HelloBug0 any news?

File: doc/zh-cn/plugins/node-status.md (hunk 4 above)
Comment (membphis) [reply doc/zh-cn/plugins/node-status.md:68]: only remove `"methods": ["PUT", "GET"],` ^_^

**Review by membphis — COMMENTED** [review]
LGTM, except minor things

File: doc/zh-cn/plugins/node-status.md (hunk 4 above)
Comment (HelloBug0) [reply doc/zh-cn/plugins/node-status.md:68]: OK, I will remove it now~

File: doc/zh-cn/plugins/node-status.md (hunk 1 above)
Comment (HelloBug0) [reply doc/zh-cn/plugins/node-status.md:24]: I misunderstand this comment before. I have changed it in the newest commit. Thanks for your comments : )

**Review by membphis — APPROVED** [review]

File: doc/zh-cn/plugins/node-status.md [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+#
+-->
+
+
+
+# 目录
+
+- [**插件简介**](#插件简介)
+- [**插件属性**](#插件属性)
+- [**插件接口**](#接口插件)
```
Comment (juzhiyuan) [inline doc/zh-cn/plugins/node-status.md:26]: There should be `插件接口`

File: doc/zh-cn/plugins/node-status.md [hunk 10]
```diff
@@ -0,0 +1,142 @@
...
+- [**启用插件**](#启用插件)
+- [**测试插件**](#测试插件)
+- [**禁用插件**](#禁用插件)
+
+
+## 插件简介
+
+`node-status` 是 `APISIX` 的请求状态查询插件，返回基本的状态信息。
```
Comment (juzhiyuan) [inline doc/zh-cn/plugins/node-status.md:34]:
```suggestion
`node-status` 是 `APISIX` 的请求状态查询插件，返回基本的状态信息。
```

File: doc/zh-cn/plugins/node-status.md [hunk 11]
```diff
@@ -0,0 +1,142 @@
...
+
+
+## 插件属性
+
+插件属性为空。
```
Comment (juzhiyuan) [inline doc/zh-cn/plugins/node-status.md:39]: 无

File: doc/zh-cn/plugins/node-status.md [hunk 12]
```diff
@@ -0,0 +1,142 @@
...
+## 插件接口
+
+插件增加接口 `/apisix/status`，可通过 [interceptors](plugin-interceptors.md) 保护该接口。
+
+
+## 启用插件
+
+1. 配置文件 `apisix/conf/config.yaml` 的 plugin list 中配置 `node-status`
+
+```
```
Comment (juzhiyuan) [inline doc/zh-cn/plugins/node-status.md:51]:
```suggestion
Not sure what this format is, cc @membphis 
```

File: doc/zh-cn/plugins/node-status.md [hunk 13]
```diff
@@ -0,0 +1,142 @@
...
+  - jwt-auth
+  - zipkin
+  ......
+```
+
+启动 `APISIX` 之后，即可访问该插件提供的接口，获得基本的状态信息。
+
+2. 在创建 route 时添加插件 `node-status`
+
+```
```
Comment (juzhiyuan) [inline doc/zh-cn/plugins/node-status.md:65]:
```
```sh
```

**Review by juzhiyuan — CHANGES_REQUESTED** [review]

File: doc/zh-cn/plugins/node-status.md (hunk 10 above)
Comment (HelloBug0) [reply doc/zh-cn/plugins/node-status.md:34]: This means?

Comment (juzhiyuan) [reply doc/zh-cn/plugins/node-status.md:34]: Oh!! My suggestion is lost, just ignored this one.

**Review by juzhiyuan — APPROVED** [review]
LGTM

**Comment by spacewander** [comment]
@HelloBug0 
Wait for your English one 😃

**Comment by juzhiyuan** [comment]
Tracking issue https://github.com/apache/apisix/issues/2379

**Pull request merged**